css/ast/src/selector.rs

```diff
@@ -108,6 +91,34 @@ pub enum AttrSelectorOp {
     Asterisk,
 }
 
+#[ast_node]
+#[derive(Is)]
+pub enum AttrSelectorValue {
+    #[tag("String")]
+    Str(Str),
+
+    #[tag("Text")]
+    Text(Text),
+}
+
+#[ast_node("AttributeSelector")]
+pub struct AttrSelector {
+    pub span: Span,
+    pub prefix: Option<Text>,
+    pub name: Text,
+    pub matcher: Option<AttrSelectorMatcher>,
```

Comment:
Is `Matcher` a word used in the specification?

Reply:
Yes

> \<attr-matcher\> = [ '~' | '|' | '^' | '$' | '*' ]? '='

https://www.w3.org/TR/selectors-4/#grammar

Reply:
Thanks for the clarification!